cli/command/service/opts.go

```diff
@@ -377,6 +403,7 @@ func (opts *serviceOptions) ToService() (swarm.ServiceSpec, error) {
 			ContainerSpec: swarm.ContainerSpec{
 				Image:    opts.image,
 				Args:     opts.args,
+				Command:  opts.entrypoint.Value(),
```

Comment:
If entrypoint is not specified on the commandline, `(*ShlexOpt).Set` won't be called, and `Value` will return `nil` here, preserving the default value, right?